priv/repo/seeds/dev/virtual_hosts_and_servers.exs

```diff
@@ -2,18 +2,19 @@ alias LiveWebServer.Repo
 alias LiveWebServer.Core
 
 owner_name_and_hostname_pairs = [
```

Comment:
Since the number of elements in the tuples is now 3 instead of 2, change the variable name to `virtual_host_tuples`.